src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -1950,38 +1950,20 @@ void ObjectSynchronizer::do_final_audit_and_print_stats() {
   log_info(monitorinflation)("Starting the final audit.");
 
   if (log_is_enabled(Info, monitorinflation)) {
-    // The other audit_and_print_stats() call is done at the Debug
-    // level at a safepoint in SafepointSynchronize::do_cleanup_tasks.
-    audit_and_print_stats(true /* on_exit */);
+    LogStreamHandle(Info, monitorinflation) ls;
+    audit_and_print_stats(&ls, true /* on_exit */);
   }
 }
 
-// This function can be called at a safepoint or it can be called when
-// we are trying to exit the VM. When we are trying to exit the VM, the
-// list walker functions can run in parallel with the other list
-// operations so spin-locking is used for safety.
```

Comment:
Maybe get rid of the spin locking comments. Otherwise this looks great!

Reply:
I updated this comment to remove the mention of spin-locking which I think it used to do.  Thanks!

Reply:
Yes, used to have a much more complicated async deflation mechanism and we had to
use spin-locking for safety. @fisk simplified this a long time ago...